tests/test_file.py

```diff
@@ -45,6 +45,18 @@ def test_locale(self, mozwebqa):
             print "There is no language selector on the page"
 
     @pytest.mark.nondestructive
+    def test_locale_zh_TW (self, mozwebqa):
+        main_page = MySiteHomePage(mozwebqa)
+        zh_TW_locale = ['/zh-TW']
+        for path in zh_TW_locale:
+            url = main_page.base_url + path
+
+            Assert.true(url.endswith('/en-US'))
+
+            if !url:
```

Comment:
The correct syntax for checking a falsey value here is `if not url`, however this will never be the case as `url` is a string and therefore is truthy.